src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ElasticSearchAnalysisGraph.java

```diff
@@ -88,13 +88,16 @@
 public class ElasticSearchAnalysisGraph extends AnalysisGraph {
 
   private static final Logger LOG = LogManager.getLogger(ElasticSearchAnalysisGraph.class);
+  private static final int rcaPeriod = 12;
```

Comment:
we should use snake case for static variables

Reply:
Done